src/cuda.jl

```diff
@@ -2,14 +2,14 @@
 const WARPSIZE = UInt32(32)
 
 macro shfl_multiple_sync(mask, target, width, vars...)
-    all_lines = map(vars) do v
-        Expr(:(=), v,
-            Expr(:call, :shfl_sync,
-                mask, v, target, width
-            )
-        )
-    end
-    return esc(Expr(:block, all_lines...))
+	all_lines = map(vars) do v
```

Comment:
Indentation is now consistent in this file but is 8 spaces, 4 would be consistent with the rest of the code.